vip-helpers/vip-media.php

```diff
@@ -78,21 +78,21 @@ function wpcom_vip_download_image( $image_url, $post_id = 0, $description = '',
  * @param string $strip What data to strip: exif|color|all
  */
 function wpcom_vip_set_image_quality( $quality, $strip = false ) {
-	add_filter( 'wp_get_attachment_url', function( $attachment_url ) use ( $quality, $strip ) {
+	add_filter( 'wp_get_attachment_url', function ( $attachment_url ) use ( $quality, $strip ) {
 		return wpcom_vip_set_image_quality_for_url( $attachment_url, $quality, $strip );
 	});
 
-	add_filter( 'the_content', function( $content ) use ( $quality, $strip ) {
+	add_filter( 'the_content', function ( $content ) use ( $quality, $strip ) {
 		if ( false !== strpos( $content, 'files.wordpress.com' ) ) {
-			$content = preg_replace_callback( '#https?://\w+\.files\.wordpress\.com[^\s"\'>]+#', function( $matches ) use ( $quality, $strip ) {
+			$content = preg_replace_callback( '#https?://\w+\.files\.WordPress\.com[^\s"\'>]+#', function ( $matches ) use ( $quality, $strip ) {
```

Comment:
This feels like a false positive and should probably be reverted.

May also want to make the regex case-insensitive as well.